maven-plugin-plugin/src/it/ant-reference-parameter/src/main/java/org/apache/maven/plugins/antWithRefs/ArtifactsTask.java

```diff
@@ -29,7 +29,7 @@
     public void execute()
         throws BuildException
     {
-        log( String.valueOf( getProject().getReference( "artifacts" ) ) );
+        log( String.valueOf( ( Object ) getProject().getReference( "artifacts" ) ) );
```

Comment:
The reference is never null?

Reply:
Without such cast we will have exception:

```
Caused by: java.lang.ClassCastException: java.util.LinkedHashSet cannot be cast to [C
    at org.apache.maven.plugins.antWithRefs.ArtifactsTask.execute (ArtifactsTask.java:32)
    at org.apache.tools.ant.UnknownElement.execute (UnknownElement.java:299)
    at sun.reflect.NativeMethodAccessorImpl.invoke0 (Native Method)
```

As scripting Mojo are deprecated I didn't investigate it, simply did in such way to make test pass.

Reply:
Yes, true. That is fully acceptable!